communication/vehicleconnections/mavsdkvehicleconnection.cpp

```diff
@@ -843,6 +843,24 @@ void MavsdkVehicleConnection::setActiveAutopilotIDOnVehicle(int id)
         qDebug() << "Warning: could not send MISSION_SET_CURRENT via MAVLINK (" << convertMavlinkPassthroughResult(result) << ")";
 }
 
+void MavsdkVehicleConnection::requestGearSwitch(int gpioPin, VehicleConnection::Gearbox gear_pwm)
+{
```

Comment:
In my opinion, ControlTower should not need to know which GPIO pin and PWM the vehicle needs to use to switch gears (it is very vehicle specific and could be an entirely different implementation). Instead I would like the interface to be sth like:
`void MavsdkVehicleConnection::requestGearSwitch(int gearNumber)`
Then, the vehicle needs to figure out how to achieve this.